bitwarden-cli/package.yml

```diff
@@ -0,0 +1,29 @@
+name       : bitwarden-cli
+version    : 2023.8.2
+release    : 1
+source     :
+    - https://github.com/bitwarden/clients/archive/refs/tags/cli-v2023.8.2.tar.gz : bf17802d24fe6b85ab74dce5bb5060ccfb5187d70f08a8473f65194dd3c21303
+license    : GPL-3.0-only
+component  : security
```

Comment:
needs `homepage:  `

Reply:
Well, I clearly followed the instructions to the letter :disappointed: